packages/cli/src/util/output/table.ts

```diff
@@ -26,10 +28,12 @@ export default function table(
     .map((size, i) => (i < margins.length && size + margins[i]) || size);
 
   // Print header
-  console.log(chalk.grey(printLine(fieldNames, sizes)));
+  output.print(chalk.grey(printLine(fieldNames, sizes)));
+  output.print('\n');
 
   // Print content
   for (const row of data) {
-    console.log(printLine(row, sizes));
+    output.print(printLine(row, sizes));
+    output.print('\n');
```

Comment:
This is a breaking change since you are changing from printing `stdout` to `stderr`

Reply:
@styfle fixed with e1096b84e